hunt_queries/llm/README.md

```diff
@@ -0,0 +1,24 @@
+# LLM Threat Hunting Queries
```

Comment:
don't feel like we need one readme.md per folder (e.g. llm), those details can fit well in the rule description and also maybe a `references` field in the toml/doc for blogs and more links.

Reply:
Noted it as optional and added the `references` field support for markdown generation.